### ISSUE TITLE
Make the operator installable without needing a go development environment

### ISSUE THREAD

**Issue by rofrano** [sustainable-computing-io/kepler-operator#46]
I tried to deploy your operator on our OpenShift cluster and got the following error:

```bash
$ make deploy IMG=quay.io/sustainable_computing_io/kepler-operator:latest
make: go: Command not found
mkdir -p /Users/rofrano/GitHub/kepler-operator/bin
test -s /Users/rofrano/GitHub/kepler-operator/bin/controller-gen || GOBIN=/Users/rofrano/GitHub/kepler-operator/bin go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.10.0
bash: go: command not found
make: *** [/Users/rofrano/GitHub/kepler-operator/bin/controller-gen] Error 127
```

It looks like only go programmers can use your operator. Unfortunately, I am not!

This Issue is to request that you publish a release of your binary here on GitHub so that non-go programmers can install and use your operator? Thanks.

**Comment by rootfs** [comment]
The `kepler-operator` is developer oriented. It will produce the end user oriented operator at OperatorHub. The process is still onoging (https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/2223).

**Comment by rofrano** [comment]
Got it! Thanks for letting me know.

I did create a `.devcontainer` folder for VSCode that spins up a Go environment in a Docker container using the [Visual Studio Code Dev Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension. I can make a pull request if you are interested. I was able to build the operator and deploy it to my cluster using it. Might be good for people without Go development environments to spin one up quickly.

**Comment by rootfs** [comment]
Sounds good. Can you make a PR and add it to https://github.com/sustainable-computing-io/kepler/tree/main/doc/dev ?

**Comment by rofrano** [comment]
Sure, here it is: #50 

I tested it with `kind` and everything seemed work fine. The environment contains go, docker-in-docker, oc, kubectl, helm, kustomize, & kind. Everything a developer should need. Let me know if you need any changes or have any questions.

**Comment by rootfs** [comment]
@rofrano looks good! You might consider blogging about it

**Comment by husky-parul** [comment]
Closing this issue post merged #50
